### PR TITLE
Ud/research

### PR DESCRIPTION
**Pull request by Udayan12167**

### CODE REVIEW

**Comment by sils** [comment]
ack https://github.com/coala/teams/pull/38/commits/41cebca730f74062764b914466d0553521d0fa76 https://github.com/coala/teams/pull/38/commits/129e45f1f5e12995e65753abd813c40914970783 https://github.com/coala/teams/pull/38/commits/340d0c4ae1d295f78de95a5b1b90352a5f313710

**Review by sils — APPROVED** [review]
ignoring the spacing problems but keep an eye out in the future

**Comment by sils** [comment]
@rultor merge

File: README.md [hunk 1]
```diff
@@ -60,8 +60,11 @@ Research
 __Team Leader:__ Udayan Tandon
 (udayan@coala.io)
 
-Work on new architecture for coala parts. Help with thesis writing and
-implementing algorithms for the science community.
+The Research team organizes and plans several aspects of coala. It is the
+team's responsibility is to draft coherent and precise proposals for
```
Comment (sims1253) [inline README.md]: double is

File: README.md [hunk 2]
```diff
@@ -60,8 +60,11 @@ Research
...
+enhancements to coala. These enhancements include architecture changes, novel
+algorithms for code analysis, and new extensions to coala.Furthermore, the team
```
Comment (sims1253) [inline README.md]: coala. <space> Furthermore

File: research/research_ideas.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+Research Team
+=============
+
+Team
+----
+
+__[Udayan Tandon](https"//github.com/udayan12167) (Leader):__
```
Comment (sims1253) [inline research/research_ideas.md]: link is broken

File: research/research_ideas.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+"I'm Udayan Tandon. I have been a part of coala since early 2015. I have seen
+coala evolve from its very primitive stages to the behemoth it is now. Mostly
+being involved in these evolutions. I'm also fresh out of university where my
+main research area was Program Analysis and my work won the best thesis award as well. I have also designed the new core of coala which is under
```
Comment (sims1253) [inline research/research_ideas.md]: line too long

**Comment by sims1253** [comment]
unack 340d0c4

**Comment by sims1253** [comment]
at least fix the link :)

**Comment by sims1253** [comment]
ack a0be74e4dcde804309764408303cbda7744fdca4
ack f7266b59afe8bd5809741de982654e032686a9ae
ack 7c865c3d41f20dabb3c10cc7ded319eed7262bc5

**Comment by sims1253** [comment]
@rultor merge

**Comment by sims1253** [comment]
@rultor stop

**Comment by sims1253** [comment]
reack aaede79
reack dc4e8ec
reack db3c3d3

**Comment by sims1253** [comment]
reack aaede79c72da93fb8c99d137a381908ab53477dd
reack dc4e8ecf1b226a7c798719c2015eb98bf8ffa268
reack db3c3d39694065a10c3ca916a4f6ad7be3ccc5a7

**Comment by sims1253** [comment]
@rultor merge

**Pull request merged**